include/ck_tile/ops/grouped_convolution/kernel/grouped_convolution_forward_kernel.hpp

```diff
@@ -998,6 +1002,14 @@ struct GroupedConvolutionForwardKernel
         OutDataType* base_c_ptr =
             static_cast<OutDataType*>(kargs.out_ptr) + group_offset_c + output_batch_offset;
 
+        // Apply group offsets to D tensors
+        std::array<const void*, NumDTensor> ds_ptr_with_offsets;
+        for(index_t d = 0; d < NumDTensor; d++)
+        {
+            ds_ptr_with_offsets[d] = static_cast<const OutDataType*>(kargs.ds_ptr[d]) +
```

Comment:
Just wander whether you shouldn't cast to DsDataType ? https://github.com/ROCm/composable_kernel/blob/e4f81586b8b5bfc6409741362418a8dbce1fa35e/include/ck_tile/ops/grouped_convolution/kernel/grouped_convolution_forward_kernel.hpp#L470C11-L470C23

Although there's a static assert when creating ds tensor view that they have to be the same as OutDataType.